spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -8,23 +8,23 @@
       Timecop.return
     end
 
-    context 'yesterday' do
+    context 'past date' do
```

Comment:
Looking at this again now. All three tests (past, present, future) are identical and we should strip this down to just one test